pkgs/development/interpreters/guile/default.nix

```diff
@@ -76,43 +89,43 @@
       "--without-threads"
     ];
 
-  postInstall = ''
-    wrapProgram $out/bin/guile-snarf --prefix PATH : "${gawk}/bin"
-  ''
+    postInstall = ''
+      wrapProgram $out/bin/guile-snarf --prefix PATH : "${gawk}/bin"
+    ''
     # XXX: See http://thread.gmane.org/gmane.comp.lib.gnulib.bugs/18903 for
     # why `--with-libunistring-prefix' and similar options coming from
     # `AC_LIB_LINKFLAGS_BODY' don't work on NixOS/x86_64.
-  + ''
-    sed -i "$out/lib/pkgconfig/guile"-*.pc    \
-        -e "s|-lunistring|-L${libunistring}/lib -lunistring|g ;
-            s|^Cflags:\(.*\)$|Cflags: -I${libunistring}/include \1|g ;
-            s|-lltdl|-L${libtool.lib}/lib -lltdl|g ;
-            s|includedir=$out|includedir=$dev|g
-            "
-  '';
-
-  # make check doesn't work on darwin
-  # On Linuxes+Hydra the tests are flaky; feel free to investigate deeper.
-  doCheck = false;
-  doInstallCheck = doCheck;
-
-  setupHook = ./setup-hook-2.2.sh;
-
-  meta = {
-    description = "Embeddable Scheme implementation";
-    homepage    = "https://www.gnu.org/software/guile/";
-    license     = lib.licenses.lgpl3Plus;
-    maintainers = with lib.maintainers; [ ludo lovek323 vrthra ];
-    platforms   = lib.platforms.all;
-
-    longDescription = ''
-      GNU Guile is an implementation of the Scheme programming language, with
-      support for many SRFIs, packaged for use in a wide variety of
-      environments.  In addition to implementing the R5RS Scheme standard
-      and a large subset of R6RS, Guile includes a module system, full access
-      to POSIX system calls, networking support, multiple threads, dynamic
-      linking, a foreign function call interface, and powerful string
-      processing.
+    + ''
+      sed -i "$out/lib/pkgconfig/guile"-*.pc    \
```

Comment:
I would prefer if this would be converted to substituteInPlace to notice regressions earlier.